LetterValue/src/main/java/com/mycompany/lettervalue/LetterValue.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.mycompany.lettervalue;
+
+/**
+ *
+ * @author lumy
+ */
+public class LetterValue {
+    public static void main(String[] args) {
+        char[] valueTable = tableCreation();
+        
+        String word = "cab";
+        int wordValue = 0;
+        
+        for(int i = 0;i<word.length();i++){
+            for(int j = 0;j<(valueTable.length);j++){
+                if(word.charAt(i)==valueTable[j]){
+                   System.out.print(valueTable[j]); 
+                   wordValue = wordValue + j + 1;
+                }
+                    
+            }
+        }
+        System.out.print(wordValue);
+    }
+    public static char[] tableCreation(){
```

Comment:
I woudd use a for loop to create all the letters in my opinion this removes a lot of not usefull lines